main.go

```diff
@@ -268,24 +275,35 @@ func (a *Agent) Signers() ([]ssh.Signer, error) {
 	return a.signers()
 }
 
+var slots = []piv.Slot{
```

Comment:
If we only include SlotAuthentication here, there should be no difference to previous.